nbs/models.tft.ipynb

```diff
@@ -15,20 +15,14 @@
    "source": [
```

Comment:
<p>Ibidem for images</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Documentation adds the caption automatically in the html from the link caption.
```[Figure1. Figure1 caption](link)```

It seems that the preview and the nbdev documentation generation do not match.